modules/apps/forms-and-workflow/calendar/calendar-service/src/main/java/com/liferay/calendar/notification/impl/NotificationUtil.java

```diff
@@ -85,7 +85,7 @@ public static String getDefaultTemplate(
 		String templatePath = CalendarServiceConfigurationUtil.get(
 			propertyName, filter);
 
-		return ContentUtil.get(
+		return StringUtil.read(
```

Comment:
this should  catch IOException and log it, to preserve the original behavior.

Reply:
Yes, I'll update this change in the next pull.